hat/assets/js/apps/Iaso/domains/orgUnits/details.js

```diff
@@ -364,11 +364,6 @@ const OrgUnitDetail = ({ params, router }) => {
         sourcesSelected,
     ]);
 
-    useEffect(() => {
```

Comment:
What was that useEffect for? It suspiciouslsy like something that was added to squash an older bug

Reply:
duno ...